source/Nabla/CIrrDeviceStub.cpp

```diff
@@ -13,6 +13,17 @@
 
 #include "CFileSystem.h"
 
+namespace nbl
+{
+	namespace video
+	{
+#ifdef _NBL_COMPILE_WITH_OPENGL_
+		core::smart_refctd_ptr<IVideoDriver> createOpenGLDriver(const nbl::SIrrlichtCreationParameters& params,
+			io::IFileSystem* io, CIrrDeviceStub* device, asset::IAssetManager* assmgr, const asset::IGLSLCompiler* glslcomp);
+#endif
+	}
+} // end namespace nbl

```

Comment:
I'm partial to not using `_NBL_COMPILE_WITH_OPENGL_` anymore (and it not being around anymore) and dynamically loading ALL the functions like in https://github.com/Hazardu/IrrlichtBAW/tree/runtimeOpenGLLoading

basically be able to handle not finding the opengl dll on the system

similar for OpenGL ES and Vulkan